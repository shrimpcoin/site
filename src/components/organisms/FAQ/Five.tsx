import React from 'react';
import styled from 'styled-components';

import { Button } from 'components/atoms/Button';
import { Link } from 'components/atoms/Link';

const Container = styled.div`
  position: relative;
  width: 100%;
  height: 100%;
  background: var(--color-dark);
`;

const Content = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-evenly;
  height: 100%;
`;

const Text = styled.div`
  positon: absolute;
  padding: 0.5em;
  white-space: pre-wrap;

  font-size: 1.5em;
  font-family: Manrope;
  font-weight: 300;
  color: white;
`;

export const Five: React.FC = (): React.ReactElement => (
  <Container>
    <Content>
      <Text style={{ borderBottom: '1px solid white', fontWeight: 400 }}>How can I support this amazing project?</Text>
      <Text style={{ width: '70%' }}>
        Firstly, you can
        <Link color="white" fadecolor="black" href="#" style={{ padding: '0px', fontWeight: 500, fontSize: 'inherit' }}>
          {' Get Started '}
        </Link>
        and buy some Shrimpcoin for yourself. Just to see how good it feels in your wallet.{'\n\n'}You can also
        <Link color="white" fadecolor="black" href="#" style={{ padding: '0px', fontWeight: 500, fontSize: 'inherit' }}>
          {' donate to 8oC'}
        </Link>
        , the organization behind Shrimpcoin. We also make all sorts of other projects and art, which you’ll probably
        like if you’ve read this far.{'\n\n'}If you’re a developer, feel free to
        <Link color="white" fadecolor="black" href="#" style={{ padding: '0px', fontWeight: 500, fontSize: 'inherit' }}>
          {' join the Shrimpcoin Discord '}
        </Link>
        or start cracking issues on our
        <Link color="white" fadecolor="black" href="#" style={{ padding: '0px', fontWeight: 500, fontSize: 'inherit' }}>
          {' GitHub repo'}
        </Link>
        .
      </Text>
    </Content>
  </Container>
);
