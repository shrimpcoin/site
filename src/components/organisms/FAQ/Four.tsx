import React from 'react';
import styled from 'styled-components';

import { ReactComponent as Geometry } from 'assets/images/faqsword.svg';

import { Button } from 'components/atoms/Button';

const Container = styled.div`
  position: relative;
  width: 100%;
  height: 100%;
  background: var(--color-light);
`;

const Content = styled.div`
  display: flex;
  position: absolute;
  align-items: center;
  justify-content: space-evenly;
  z-index: 1;
  height: 100%;
  width: 100%;
`;

const Question = styled.div`
  position: absolute;
  padding: 1em;

  z-index: 2;

  font-size: 1.5em;
  font-family: Manrope;
  font-weight: 400;
  color: black;
`;

const Answer = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-evenly;
  z-index: 1;

  font-family: 'Manrope';
  font-size: 1.5em;
  font-weight: 300;
`;

export const Four: React.FC = (): React.ReactElement => (
  <Container>
    <Geometry style={{ position: 'absolute', left: '0%', width: '100%', margin: 'auto' }} />
    <Question style={{ top: '6.5%', left: '50%', transform: 'translateX(-50%)' }}>
      ...<strong>Fine.</strong> What makes Shrimpcoin a good stablecoin?
    </Question>
    <Content>
      <Answer style={{ width: '18%' }}>
        <p style={{ borderBottom: '1px solid black', fontWeight: 400 }}>Low Gas Prices</p>
        Thanks to the work of countless Ergonauts, gas prices - or transaction fees - are very low compared to other
        platforms. Many stablecoins reside on the Ethereum blockchain, which is notorious for its high gas prices.
        <Button background="var(--color-light)" outline="black" href="#" style={{ margin: '2em' }}>
          Learn More
        </Button>
      </Answer>
      <Answer style={{ width: '18%' }}>
        <p style={{ borderBottom: '1px solid black', fontWeight: 400 }}>Open Source</p>
        Unlike other popular stablecoins, Shrimpcoin is fully open-source; even the code for this website is hosted on
        our GitHub. This means that Shrimpcoin is constantly being improved by the people that actually use it.
        <Button background="var(--color-lightest)" outline="black" href="#" style={{ margin: '2em' }}>
          Learn More
        </Button>
      </Answer>
      <Answer style={{ width: '18%' }}>
        <p style={{ borderBottom: '1px solid black', fontWeight: 400 }}>Shrimp Technology</p>
        Our shrimp roots enable us to shrimp faster and harder than any other cryptocurrency on the market. By utilizing
        our shrimp techology, we can optimize our shellfish to a level only theorized by some.
        <Button background="var(--color-light)" outline="black" href="#" style={{ margin: '2em' }}>
          Learn More
        </Button>
      </Answer>
    </Content>
  </Container>
);
