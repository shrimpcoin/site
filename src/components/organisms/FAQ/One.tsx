import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
  position: relative;
  width: 100%;
  height: 100%;
  background: var(--color-primary);
`;

const Content = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-evenly;
`;

const Text = styled.div`
  positon: absolute;
  padding: 0.5em;

  font-size: 1.5em;
  font-family: Manrope;
  font-weight: 200;
  color: white;

  border-bottom: 1px solid white;
`;

export const One: React.FC = (): React.ReactElement => (
  <Container>
    <Content>
      <Text>
        What <strong>is</strong> Shrimpcoin?
      </Text>
      <Text style={{ padding: '1em', border: '2px solid white', width: '40%' }}>
        <strong>We're glad you asked.</strong> Shrimpcoin (SHMPC) is a revolutionary new stable cryptocurrency that is
        pinned to the value of exactly one shrimp.
      </Text>
    </Content>
  </Container>
);
