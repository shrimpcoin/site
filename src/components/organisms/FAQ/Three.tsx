import React from 'react';
import styled from 'styled-components';

import { Button } from 'components/atoms/Button';

const Container = styled.div`
  position: relative;
  width: 100%;
  height: 100%;
  background: var(--color-lightest);
`;

const Content = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-evenly;
  height: 100%;
`;

const Text = styled.div`
  positon: absolute;
  padding: 0.5em;
  white-space: pre-wrap;

  font-size: 1.5em;
  font-family: Manrope;
  font-weight: 300;
  color: black;
`;

export const Three: React.FC = (): React.ReactElement => (
  <Container>
    <Content>
      <Text style={{ borderBottom: '1px solid black', fontWeight: 400 }}>You didn't answer my question.</Text>
      <Text style={{ width: '70%' }}>
        {
          'Shrimpcoin is backed by SCHIT (ShrimpCoin Hoard Index Token), which provides liquidity to Shrimpcoin HODLers.\n\nThe value of SCHIT goes up when ERG goes up, and down when ERG goes down, because more ERG is needed to cover the outstanding Shrimpcoin.\n\nSCHIT price also increases when people use the Shrimpcoin contract, as a portion of the transaction fees collected goes to the value of SCHIT.'
        }
      </Text>
      <Button background="var(--color-lightest)" outline="black" href="#">
        Learn More
      </Button>
    </Content>
  </Container>
);
