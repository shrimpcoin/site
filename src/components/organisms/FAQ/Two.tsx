import React from 'react';
import styled from 'styled-components';

import { ReactComponent as Geometry } from 'assets/images/roundedtriangle.svg';

const Container = styled.div`
  position: relative;
  width: 100%;
  height: 100%;
  background: var(--color-light);
`;

const Content = styled.div`
  display: flex;
  position: absolute;
  align-items: center;
  justify-content: space-evenly;
  z-index: 2;
  height: 100%;
`;

const Text = styled.div`
  positon: absolute;
  padding: 1em;

  font-size: 1.5em;
  font-family: Manrope;
  font-weight: 300;
  color: black;
`;

export const Two: React.FC = (): React.ReactElement => (
  <Container>
    <Geometry style={{ position: 'absolute', left: '0px', height: '100%', width: '40%' }} preserveAspectRatio="none" />
    <Content>
      <Text style={{ fontWeight: 400 }}>Why shrimp?</Text>
      <Text style={{ width: '40%' }}>
        The cutting-edge Shrimpcoin protocol runs on the Ergo blockchain, using smart contracts to determine the price
        of shrimp on the fly.
      </Text>
    </Content>
  </Container>
);
