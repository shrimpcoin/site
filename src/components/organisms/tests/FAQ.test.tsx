import React from 'react';
import renderer from 'react-test-renderer';
import { One } from 'components/organisms/FAQ/One';
import { Two } from 'components/organisms/FAQ/Two';
import { Three } from 'components/organisms/FAQ/Three';
import { Four } from 'components/organisms/FAQ/Four';
import { Five } from 'components/organisms/FAQ/Five';

test('One', () => {
  const component = renderer.create(<One />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Two', () => {
  const component = renderer.create(<Two />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Three', () => {
  const component = renderer.create(<Three />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Four', () => {
  const component = renderer.create(<Four />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Five', () => {
  const component = renderer.create(<Five />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
