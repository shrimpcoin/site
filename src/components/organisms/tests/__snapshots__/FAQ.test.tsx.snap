// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Five 1`] = `
<div
  className="sc-bqGGPW hJXmDU"
>
  <div
    className="sc-ksluID iFWacx"
  >
    <div
      className="sc-hBMUJo gAKgWR"
      style={
        Object {
          "borderBottom": "1px solid white",
          "fontWeight": 400,
        }
      }
    >
      How can I support this amazing project?
    </div>
    <div
      className="sc-hBMUJo gAKgWR"
      style={
        Object {
          "width": "70%",
        }
      }
    >
      Firstly, you can
      <a
        className="sc-dIsUp cdsXI"
        color="white"
        href="#"
        style={
          Object {
            "fontSize": "inherit",
            "fontWeight": 500,
            "padding": "0px",
          }
        }
      >
         Get Started 
      </a>
      and buy some Shrimpcoin for yourself. Just to see how good it feels in your wallet.
      


      You can also
      <a
        className="sc-dIsUp cdsXI"
        color="white"
        href="#"
        style={
          Object {
            "fontSize": "inherit",
            "fontWeight": 500,
            "padding": "0px",
          }
        }
      >
         donate to 8oC
      </a>
      , the organization behind Shrimpcoin. We also make all sorts of other projects and art, which you’ll probably like if you’ve read this far.
      


      If you’re a developer, feel free to
      <a
        className="sc-dIsUp cdsXI"
        color="white"
        href="#"
        style={
          Object {
            "fontSize": "inherit",
            "fontWeight": 500,
            "padding": "0px",
          }
        }
      >
         join the Shrimpcoin Discord 
      </a>
      or start cracking issues on our
      <a
        className="sc-dIsUp cdsXI"
        color="white"
        href="#"
        style={
          Object {
            "fontSize": "inherit",
            "fontWeight": 500,
            "padding": "0px",
          }
        }
      >
         GitHub repo
      </a>
      .
    </div>
  </div>
</div>
`;

exports[`Four 1`] = `
<div
  className="sc-jrsJWt bBOhxk"
>
  <svg
    style={
      Object {
        "left": "0%",
        "margin": "auto",
        "position": "absolute",
        "width": "100%",
      }
    }
  >
    faqsword.svg
  </svg>
  <div
    className="sc-iqAclL ibTngR"
    style={
      Object {
        "left": "50%",
        "top": "6.5%",
        "transform": "translateX(-50%)",
      }
    }
  >
    ...
    <strong>
      Fine.
    </strong>
     What makes Shrimpcoin a good stablecoin?
  </div>
  <div
    className="sc-kEqXSa lfQcTh"
  >
    <div
      className="sc-crzoAE gbrlJu"
      style={
        Object {
          "width": "18%",
        }
      }
    >
      <p
        style={
          Object {
            "borderBottom": "1px solid black",
            "fontWeight": 400,
          }
        }
      >
        Low Gas Prices
      </p>
      Thanks to the work of countless Ergonauts, gas prices - or transaction fees - are very low compared to other platforms. Many stablecoins reside on the Ethereum blockchain, which is notorious for its high gas prices.
      <a
        className="sc-gKAaRy bnVQKL"
        href="#"
        style={
          Object {
            "margin": "2em",
          }
        }
      >
        Learn More
      </a>
    </div>
    <div
      className="sc-crzoAE gbrlJu"
      style={
        Object {
          "width": "18%",
        }
      }
    >
      <p
        style={
          Object {
            "borderBottom": "1px solid black",
            "fontWeight": 400,
          }
        }
      >
        Open Source
      </p>
      Unlike other popular stablecoins, Shrimpcoin is fully open-source; even the code for this website is hosted on our GitHub. This means that Shrimpcoin is constantly being improved by the people that actually use it.
      <a
        className="sc-gKAaRy kKFXfJ"
        href="#"
        style={
          Object {
            "margin": "2em",
          }
        }
      >
        Learn More
      </a>
    </div>
    <div
      className="sc-crzoAE gbrlJu"
      style={
        Object {
          "width": "18%",
        }
      }
    >
      <p
        style={
          Object {
            "borderBottom": "1px solid black",
            "fontWeight": 400,
          }
        }
      >
        Shrimp Technology
      </p>
      Our shrimp roots enable us to shrimp faster and harder than any other cryptocurrency on the market. By utilizing our shrimp techology, we can optimize our shellfish to a level only theorized by some.
      <a
        className="sc-gKAaRy bnVQKL"
        href="#"
        style={
          Object {
            "margin": "2em",
          }
        }
      >
        Learn More
      </a>
    </div>
  </div>
</div>
`;

exports[`One 1`] = `
<div
  className="sc-bdnxRM ecnezf"
>
  <div
    className="sc-gtsrHT hVIRaV"
  >
    <div
      className="sc-dlnjwi htCZdS"
    >
      What 
      <strong>
        is
      </strong>
       Shrimpcoin?
    </div>
    <div
      className="sc-dlnjwi htCZdS"
      style={
        Object {
          "border": "2px solid white",
          "padding": "1em",
          "width": "40%",
        }
      }
    >
      <strong>
        We're glad you asked.
      </strong>
       Shrimpcoin (SHMPC) is a revolutionary new stable cryptocurrency that is pinned to the value of exactly one shrimp.
    </div>
  </div>
</div>
`;

exports[`Three 1`] = `
<div
  className="sc-iCoGMd hmMEJX"
>
  <div
    className="sc-fujyAs fUZsWb"
  >
    <div
      className="sc-pNWdM zeIhq"
      style={
        Object {
          "borderBottom": "1px solid black",
          "fontWeight": 400,
        }
      }
    >
      You didn't answer my question.
    </div>
    <div
      className="sc-pNWdM zeIhq"
      style={
        Object {
          "width": "70%",
        }
      }
    >
      Shrimpcoin is backed by SCHIT (ShrimpCoin Hoard Index Token), which provides liquidity to Shrimpcoin HODLers.

The value of SCHIT goes up when ERG goes up, and down when ERG goes down, because more ERG is needed to cover the outstanding Shrimpcoin.

SCHIT price also increases when people use the Shrimpcoin contract, as a portion of the transaction fees collected goes to the value of SCHIT.
    </div>
    <a
      className="sc-gKAaRy kKFXfJ"
      href="#"
    >
      Learn More
    </a>
  </div>
</div>
`;

exports[`Two 1`] = `
<div
  className="sc-hKFxyN iGpJeU"
>
  <svg
    preserveAspectRatio="none"
    style={
      Object {
        "height": "100%",
        "left": "0px",
        "position": "absolute",
        "width": "40%",
      }
    }
  >
    roundedtriangle.svg
  </svg>
  <div
    className="sc-eCApnc eiARsg"
  >
    <div
      className="sc-jSFjdj utkDI"
      style={
        Object {
          "fontWeight": 400,
        }
      }
    >
      Why shrimp?
    </div>
    <div
      className="sc-jSFjdj utkDI"
      style={
        Object {
          "width": "40%",
        }
      }
    >
      The cutting-edge Shrimpcoin protocol runs on the Ergo blockchain, using smart contracts to determine the price of shrimp on the fly.
    </div>
  </div>
</div>
`;
